cmd/tlin/main.go

```diff
@@ -64,19 +65,19 @@ func main() {
 
 	if config.CFGAnalysis {
 		runWithTimeout(ctx, func() {
-			runCFGAnalysis(ctx, logger, config.Paths, config.FuncName)
+			runCFGAnalysis(ctx, logger, config.Paths, config.FuncName, config.Output)
 		})
 	} else if config.CyclomaticComplexity {
 		runWithTimeout(ctx, func() {
-			runCyclomaticComplexityAnalysis(ctx, logger, config.Paths, config.CyclomaticThreshold, config.JsonOutput)
+			runCyclomaticComplexityAnalysis(ctx, logger, config.Paths, config.CyclomaticThreshold, config.JsonOutput, config.Output)
 		})
 	} else if config.AutoFix {
 		runWithTimeout(ctx, func() {
 			runAutoFix(ctx, logger, engine, config.Paths, config.DryRun, config.ConfidenceThreshold)
 		})
 	} else {
 		runWithTimeout(ctx, func() {
-			runNormalLintProcess(ctx, logger, engine, config.Paths, config.JsonOutput)
+			runNormalLintProcess(ctx, logger, engine, config.Paths, config.JsonOutput, config.Output)
 		})
 	}
 }
```

Comment:
The output file might not be created depending on the postion of -o flag.

For example, the following command correctly generates an svg file:

```bash
$ tlin -cfg -o output.svg -func highComplexity testdata/complexity/high.gno
```

But if the position of the -o flag changes, this command will only output the result to the terminal without creating a file:

```bash
$ tlin -cfg -func highComplexity testdata/complexity/high.gno -o output.svg
```

In other words, it would be better to modify the implementation to check the config.Output condition first before executing each task.

Also, since the flags have been modified, please update the readme as well.

Reply:
I'm planning to huge fix-up for CLI part soon anyway, so it's not a big problem. I'll leave this for later